typescript/packages/subsurface-viewer/src/layers/wells/utils/spline.ts

```diff
@@ -107,6 +107,13 @@ export function CatmullRom(
     return [x, y, z] as Position3D;
 }
 
+// Generate a range of numbers with given step length.
+const range = (start: number, stop: number, step: number) =>
```

Comment:
Is this the same as lodash's [`range` method?](https://lodash.com/docs/#range)

Reply:
Good point